src/plugins/ui_actions/public/components/action_wizard/index.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../../../../src/legacy/ui/public/styles/_styling_constants';
+
+.uiActions__selectedActionFactoryContainer {
+  background-color: $euiColorLightestShade;
+  padding: $euiSize;
+}
+
+.uiActions__ActionFactory {
+  width: 120px;
+  min-height: 100px;
```

Comment:
not sure about this absolutes. 
1. width needs to be absolute to have grid like effect of these items
2. min-height is needed to make sure the row is not too narrow if just 1 item with not much content in it wrapped to next line

Reply:
Usually when we need pixel values we use at least some sort of calculation with the EUI variables. 

1. If you want an actual grid, would `EuiFlexGrid` be a better choice than `EuiFlexGroup`? 
2. Perhaps @andreadelrio can help you come up with a good calculation for this min-height based on the type of content it can hold.

Reply:
@cchaos, thanks! Will rely at least rely on $euiSize

1. What is the difference between using `EuiFlexGrid` and `EuiFlexGroup` in my case? I see that `EuiFlexGrid` uses `flex` under the hood. I also don't know number of columns up-front. 

2. @andreadelrio would appreciate any input 👍

Reply:
👍 

1. I would assume that you don't want a neverending number of columns, the content will get way too narrow. EuiFlexGrid allows you to set the 'ideal' number of columns (a set amount) and then it will properly wrap the items so you have a proper grid (columns and rows).

Reply:
1. Then I believe we would want to have here at least `5` (I believe even more would fit on larger screen for `medium` fly-out) and seems like max 4 supported by `EuiFlexGrid`

Reply:
👍 Ok, then EuiFlexGroup should be fine. Again, get with @andreadelrio to come up with the ideal width.